contracts/core/VaultPriceFeed.sol

```diff
@@ -16,7 +15,7 @@ contract VaultPriceFeed is IVaultPriceFeed, Governable {
 
     constructor() {}
 
-    function setTokenConfig(address _token, address _priceFeed, uint256 _priceDecimals) external override onlyGov {
+    function setTokenConfig(address _token, address _priceFeed, uint256 _priceDecimals) external override {
```

Comment:
Do we need onlyOwner here since onlyGov is going away?

Reply:
Honestly, once we initialize the config for each assert, we dont need owner role. because nobody can update once it is initialized. so removed the owner modifier

Reply:
suggest adding back onlyOwner, malicious user can watch deploy new ERC20 from us, and front-run our setTokenConfig tx to set a malicious priceFeed